listenbrainz/webserver/views/user.py

```diff
@@ -231,7 +231,7 @@ def get_track_id_from_listen(listen):
 
 def delete_user(musicbrainz_id):
     """ Delete a user from ListenBrainz completely.
-    First, drops the user's influx measurement and then deletes her from the
+    First, drops the user's influx measurement and then deletes him/her from the
```

Comment:
given that you're fixing this anyways, we've decided to avoid gendered pronouns. Let's fix this to be `them` or `the user`.